src/main/java/core/mate/academy/service/ExcavatorProducer.java

```diff
@@ -0,0 +1,17 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import core.mate.academy.model.Machine;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer<Machine> {
+    @Override
+    public List<Machine> get() {
```

Comment:
The return type of the `get` method should be `List<Excavator>` instead of `List<Machine>`. This change is necessary to match the parameterization of the `MachineProducer` interface with `Excavator` as suggested above.